tagstore/settings/base.py

```diff
@@ -0,0 +1 @@
+FREEFORM_TAXONOMY_UID = -100
```

Comment:
Please add a comment explaining what this is :)

I also think that it would be fine to put this in a `constants.py` file rather than considering it a setting. (Why would someone need to change it? And if they did, couldn't that cause problems if it was already in use?)